posthog/warehouse/api/external_data_source.py

```diff
@@ -867,8 +868,19 @@ def source_prefix(self, request: Request, *arg: Any, **kwargs: Any):
     @action(methods=["GET"], detail=True)
     def jobs(self, request: Request, *arg: Any, **kwargs: Any):
         instance: ExternalDataSource = self.get_object()
+        after = request.query_params.get("after", None)
```

Comment:
there's a pagination/paginator pattern for viewset classes. Maybe applicable to an action? Not sure though

Reply:
Yeah, I did see this, but couldn't see any examples of it applying to actions annoyingly